cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,118 @@ describe("Timeline", () => {
+                cy.get(".mx_SearchBar_input input").clear().invoke("val", string).trigger("input").type("{enter}");
+                cy.get(".mx_EventTile:not(.mx_EventTile_contextual) .mx_EventTile_searchHighlight").should("exist");
+            };
+
+            // Open roomId room
+            cy.visit("/#/room/" + roomId);
+
+            // Wait until configuration is finished
+            cy.contains(
+                ".mx_RoomView_body .mx_GenericEventListSummary .mx_GenericEventListSummary_summary",
+                "created and configured the room.",
+            ).should("exist");
+
+            // Create a bot "BotBob" and invite it
+            let bot: MatrixClient;
+            cy.getBot(homeserver, {
+                displayName: "BotBob",
+                autoAcceptInvites: false,
+            }).then((_bot) => {
+                bot = _bot;
+                cy.inviteUser(roomId, bot.getUserId());
+                bot.joinRoom(roomId);
+
+                // Make sure the bot joined the room
+                cy.contains(
+                    ".mx_GenericEventListSummary .mx_EventTile_info.mx_EventTile_last",
+                    "BotBob joined the room",
+                ).should("exist");
+
+                // Have bot send MESSAGE to roomId
+                cy.botSendMessage(bot, roomId, MESSAGE);
+
+                // Have bot send stringJa to roomId
+                cy.botSendMessage(bot, roomId, stringJa);
+            });
+
+            // Send strings to the room
+            const strings = [stringAr, stringEl, stringHe, stringHi, stringJa, stringKo, stringTh, stringZh];
+            strings.forEach((strings) => {
+                sendString(strings);
+            });
+
+            // Send some random messages in English
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.text" as MsgType,
+                body:
+                    "Paragraph above lists\n\n" +
+                    "- item 1\n- item 2\n- item 3\n\n" +
+                    "Paragraph between lists\n\n" +
+                    "1. item 1\n2. item 2\n3. item 3\n\n" +
+                    "Paragraph below lists",
+                format: "org.matrix.custom.html",
+                formatted_body:
+                    "<p>Paragraph above lists</p>\n" +
+                    "<ul>\n<li>item 1</li>\n<li>item 2</li>\n<li>item 3</li>\n</ul>\n" +
+                    "<p>Paragraph between lists</p>\n" +
+                    "<ol>\n<li>item 1</li>\n<li>item 2</li>\n<li>item 3</li>\n</ol>\n" +
+                    "<p>Paragraph below lists</p>\n",
+            });
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.emote" as MsgType,
+                body: "sends an emote",
+            });
+            cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                msgtype: "m.text" as MsgType,
+                body: `${MESSAGE} by Alan`,
+            });
```

Comment:
why?

Reply:
To ensure search random strings can be found. ok to remove

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/10319/commits/e64b3de23f46025e26fd8ea213767569b10c1ec2